checkout.html

```diff
@@ -1,118 +1,175 @@
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+
+                <label for="Street Address">STREET ADDRESS</label>
+                <div class="input-control twoColumns">
+                    <input type="text" id="address" placeholder="STREET ADDRESS .." />
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+
+
+                <label for="City">CITY</label>
+                <div class="input-control">
+                    <select name="city" class="cities" id="cityId">
+                        <option value="">Select City</option>
+                    </select>
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+                <div class="hide"></div>
+
+                <label for="State">STATE</label>
+                <div class="input-control">
+                    <select name="state" class="states" id="stateId">
+                        <option value="">Select State</option>
+                    </select>
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+                <div class="hide"></div>
+
+                <label for="country">COUNTRY</label>
+                <div class="input-control">
+                    <select name="country" class="countries" id="countryId">
+                        <option value="">Select Country</option>
+                    </select>
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+                <div class="hide"></div>
+
+                <label for="Postal code">POSTAL CODE</label>
+                <div class="input-control">
+                    <input type="text" id="postCode" placeholder="POSTAL CODE .." />
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+                <div></div>
+
+                <label for="phone">PHONE</label>
+                <div class="input-control">
+                    <input type="text" id="phone" placeholder="PHONE .." />
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+                <div class="hide"></div>
+            </div>
+            <input type="Submit" value="Submit" id="Submit" />
+            <hr />
```

Comment:
I notice some of the text get cut off at around tablet size. May want to fix the styling but it is not urgent.

Reply:
if I could not solve the issue of cutting off texts tonight, I will create a new issue for it.